src/main/java/org/jabref/gui/desktop/os/Linux.java

```diff
@@ -16,27 +18,48 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@AllowedToUseAwt("Requires AWT to open a file with the native method")
 public class Linux implements NativeDesktop {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Linux.class);
 
+    private void nativeOpenFile(String filePath) {
+        new Thread(() -> {
```

Comment:
```suggestion
        JabRefExecutorService.INSTANCE.execute(() -> {
```

Reply:
This wouldn't close the PDF right?

Reply:
No, the behaviour should be the same, it's just that it's using a Thraed from the Threadpool instead of a new one.